pkg/pillar/hypervisor/xen.go

```diff
@@ -497,21 +497,21 @@ func (ctx xenContext) Info(domainName string, domainID int) (int, DomState, erro
 	lastState := domainState[len(domainState)-1:]
 	log.Debugf("Info: domain: %s lastState: %s", domainName, lastState)
 	//if domainState is not 'r' or 'b' then the domain is not healthy.
-	stateMap := map[string]DomState{
-		"r": Running,
-		"b": Blocked,
-		"p": Paused,
-		"s": Exiting,
-		"c": Crashed,
-		"d": Dying,
+	stateMap := map[string]types.SwState{
+		"r": types.RUNNING,
+		"b": types.RUNNING,
+		"p": types.HALTED,
+		"s": types.HALTING,
+		"c": types.BROKEN,
+		"d": types.HALTING,
 	}
 	effectiveDomainState, matched := stateMap[lastState]
 	if !matched {
-		effectiveDomainState = Unknown
+		effectiveDomainState = types.UNKNOWN
 	}
 
 	// if we are in one of the states that may require a device model -- check for it
-	if effectiveDomainState == Running || effectiveDomainState == Blocked || effectiveDomainState == Paused {
+	if effectiveDomainState == types.RUNNING || effectiveDomainState == types.HALTED {
```

Comment:
Why would HALTED require a device model?

Reply:
Again -- I'm not sure what you're reviewing @eriknordmark -- in the PR I'm looking at it is types.HALTING